icelam/html-inline-script-webpack-plugin
# feat: new option for defining HTML templates the plugin should process

## Description

Add new option to allow user defining a regex pattern to control which HTML template the plugin should handle.
Resolve #335.

## How has this been tested?

1. Add new test case
2. Ensure old test passes

## Types of changes

- [ ] Test cases - `test`
- [ ] Other(s): 

## Remarks

N/A
